index.html

```diff
@@ -112,24 +112,24 @@ <h3 class="workgridp1">Website Portfolio</h3>
             </div>
             <div class="workgrid12">
                 <div class="workgrid4">
-                    <h3 class="workgridp1">Profesional Art Printing Data</h3>
+                    <h3 class="workgridp1 title">Profesional Art Printing Data</h3>
                     <p class="workgridp2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
                     <ul>
-                        <li><p>html</p></li>
-                        <li><p>bootstrap</p></li>
-                        <li><p>Ruby</p></li>
+                        <li><p class="language1">html</p></li>
+                        <li><p class="language2">bootstrap</p></li>
+                        <li><p class="language3">Ruby</p></li>
                     </ul>
-                    <button type="button">See Project</button>
+                    <button data-modal-target="#modal-pop-up" type="button">See Project</button>
                 </div>
                 <div class="workgrid4">
-                    <h3 class="workgridp1">Profesional Art Printing Data</h3>
+                    <h3 class="workgridp1 title">Profesional Art Printing Data</h3>
                     <p class="workgridp2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
                     <ul>
-                        <li><p>html</p></li>
-                        <li><p>bootstrap</p></li>
-                        <li><p>Ruby</p></li>
+                        <li><p class="language1">html</p></li>
```

Comment:
- [ ] For this milestone, projects are displayed dynamically using javascript code, remove all code for all projects in the html file.